sunpy/conftest.py

```diff
@@ -14,27 +16,19 @@
 else:
     matplotlib.use('Agg')
 
-import sunpy.tests.helpers
-from sunpy.tests.hash import HASH_LIBRARY_NAME
-from sunpy.tests.helpers import new_hash_library, generate_figure_webpage
-from sunpy.extern import six
-
-import pytest
 
+if importlib.util.find_spec('asdf') is not None:
+    from asdf import __version__ as asdf_version
+    # Skip schema tests on astropy LT 3.1 because the coordinate schema only
+    # work for 3.1 The tags are already not registered on LT 3.1
+    from astropy import __version__ as astropy_version
+    if asdf_version >= '2.0.0' and astropy_version >= "3.1.dev0":
```

Comment:
What if astropy 3.1 releases? This check will fail on 3.1, right?

Reply:
No, this should match every version pre release or otherwise that is after the first 3.1 dev commit. It's based on the distutils version parsing machinery. adding `.dev0` is the lowest denomination of pre-release, so it matches all releases and pre-releases in the 3.1 series and then all releases after iirc.